easybuild/easyconfigs/t/tbb/tbb-2019_U9-GCCcore-8.3.0.eb

```diff
@@ -1,18 +1,15 @@
 name = 'tbb'
 version = '2019_U9'
 
-homepage = 'https://github.com/oneapi-src/oneTBB'
+homepage = 'https://01.org/tbb/'
 description = """Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
  take full advantage of multicore performance, that are portable, composable and have future-proof scalability."""
 
 toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
 
-source_urls = ['https://github.com/oneapi-src/oneTBB/archive/']
+source_urls = ['https://github.com/01org/tbb/archive/']
```

Comment:
This just just redirects to the page you switched from https://github.com/oneapi-src/oneTBB ?